mmengine/hooks/runtime_info_hook.py

```diff
@@ -128,4 +152,7 @@ def after_test_epoch(self,
         """
         if metrics is not None:
             for key, value in metrics.items():
-                runner.message_hub.update_scalar(f'test/{key}', value)
+                if _is_scalar(value):
+                    runner.message_hub.update_scalar(f'test/{key}', value)
+                else:
+                    runner.logger.info(f'{key}:\n{value}')
```

Comment:
How about using `runner.message_hub.update_info` if the value is not a scalar?

Reply:
It won't be printed in `LoggerHook`.

Reply:
Updated